docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+services:
+  rabbitmq:
+    image: rabbitmq:3
+  sos-journaler:
+    image: sync-or-swim/sos-journaler
+    command: >-
+      --rabbitmq-queue-name "fixm"
+      --rabbitmq-host "rabbitmq"
+    restart: unless-stopped
+  sos-swim-consumer:
+    image: sync-or-swim/sos-swim-consumer
+    command: >-
+      --swim-broker-url "tcps://ems2.swim.faa.gov:55443"
+      --swim-queue "xaviosx.gmail.com.FDPS.d10fdb36-40e9-4e7e-9881-57949c04112e.OUT"
```

Comment:
Hardcoding my own personal SWIM info in here is not ideal. Should these read from environment variables?

Reply:
Environment variables sound good